lib/compress/zbuff_compress.c

```diff
@@ -113,20 +112,18 @@ ZBUFF_CCtx* ZBUFF_createCCtx_advanced(ZSTD_customMem customMem)
     {
         zbc = (ZBUFF_CCtx*)calloc(1, sizeof(ZBUFF_CCtx));
         if (zbc==NULL) return NULL;
-        zbc->customAlloc = malloc;
-        zbc->customFree = free;
+        memcpy(&zbc->customMem, &defaultCustomMem, sizeof(ZSTD_customMem));
         zbc->zc = ZSTD_createCCtx();
         return zbc;
     }
 
     if (!customMem.customAlloc || !customMem.customFree)
         return NULL;
 
-    zbc = (ZBUFF_CCtx*)customMem.customAlloc(sizeof(ZBUFF_CCtx));
+    zbc = (ZBUFF_CCtx*)customMem.customAlloc(customMem.opaque, sizeof(ZBUFF_CCtx));
```

Comment:
There is now this new `customMem.opaque` in many places.
What is it for ? In which way does it help ?